<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>excelRecordManger</class>
 <widget class="QWidget" name="excelRecordManger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>excelRecordManger</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="currentTimeLabel">
       <property name="text">
        <string>当前时间：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TimeTotalLabel">
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pointTotalLabel">
       <property name="text">
        <string>总计记录数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="recordNumLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="dataID">
         <property name="text">
          <string>数据id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dataIDComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="dataNameLabel">
         <property name="text">
          <string>数据命名</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dataNameEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopRecordButton">
       <property name="text">
        <string>停止记录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
